bundles/org.openhab.binding.edf/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="edf" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="base">
+		<label>Base Tariff</label>
+		<description>Default Tariff Option</description>
+
+		<channels>
+			<channel id="tariff-start" typeId="timestamp"/>
+			<channel id="fixed-ht" typeId="price-ht">
+				<label>Fix Price HT</label>
+				<description>Yearly fix price excluding taxes.</description>
+			</channel>
+			<channel id="fixed-ttc" typeId="price-ttc">
+				<label>Fix Price</label>
+				<description>Yearly fix price including taxes.</description>
+			</channel>
+			<channel id="variable-ht" typeId="energy-price-ht">
```

Comment:
What is the difference between fixed-ht and variable-ht ?
There is no variability in EDF tariffs ?

Reply:
The fixed part is the "abonnement", variable is per kWh